.github/workflows/tests.yml

```diff
@@ -247,7 +247,7 @@ jobs:
       - name: Install brew tests dependencies
         run: |
           brew install subversion
-          Library/Homebrew/shims/scm/svn --homebrew=print-path
+          brew sh -c "svn --homebrew=print-path"
```

Comment:
The previous behaviour, to my knowledge, is not supported? I hope no one relied on being able to invoke a shim outside of the `brew` environment.

Reply:
Given that we were in our CI environment: I don't think we should break this. At least it'd be good to attempt to infer `HOMEBREW_LIBRARY` if it's not specified based on the shim location?

Reply:
Tricky because of the different directory levels. Probably would need to go through levels and search for a specific file/folder.

Note that any `HOMEBREW_MACOS_VERSION_NUMERIC` check in this file would have also not worked, and never has when called this way.

Reply:
Fine with me then. Can we try to produce a human-readable error when this isn't defined? Also, I reckon hold off until after 3.0.2 is tagged so we can get a bit of wider testing time with this before a new tag.